pbm/node.go

```diff
@@ -106,6 +106,28 @@ func (n *Node) GetInfo() (*NodeInfo, error) {
 	return i, nil
 }
 
+// MaxDBSize returns the size of the largest database on replicaset.
+// It is the of files on disk in bytes
```

Comment:
Why just the largest db? When we use mongodump with --archive all the database namespaces go into one file.

Reply:
Yep, my glitch